pkg/cli/admin/ocpcertificates/regeneratemco/updatesecrets.go

```diff
@@ -0,0 +1,105 @@
+package regeneratemco
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/vincent-petithory/dataurl"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+func (o *RegenerateMCOOptions) RunUserDataUpdate(ctx context.Context) error {
+	return o.updateSecrets(ctx)
+}
+
+func (o *RegenerateMCOOptions) updateSecrets(ctx context.Context) error {
+	clientConfig, err := o.RESTClientGetter.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+	clientset, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return err
+	}
+
+	// Get the updated CA cert
+	mcoSecrets := clientset.CoreV1().Secrets(mcoNamespace)
+	mcsSecret, err := mcoSecrets.Get(ctx, newMCSCASecret, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("cannot read MCS secret: %w", err)
+	}
+
+	caCert, exists := mcsSecret.Data[corev1.TLSCertKey]
+	if !exists {
+		return fmt.Errorf("could not find MCS CA cert at %s after rotation", corev1.TLSCertKey)
```

Comment:
```suggestion
		return fmt.Errorf("could not find MCS CA cert at %s", corev1.TLSCertKey)
```
Maybe the user did not perform the rotation yet.

Also, the `%s` is not helpful here, you'll want to call out the location of the secret, too.